backend/codesirius/problems/serializers/problem.py

```
@@ -0,0 +1,113 @@
+
+    def validate_tags(self, tags):
+        if not tags:
+            raise serializers.ValidationError("Minimum one tag is required.")
+        return tags
+
+    def validate_languages(self, languages):
+        if not languages:
+            raise serializers.ValidationError("Minimum one language is required.")
+        return languages
+
+    def create(self, validated_data):
```

Ruff (0.8.2)

Missing return type annotation for public function `create`

(ANN201)